src/python/analyze_spiget.py

```diff
@@ -1,11 +1,8 @@
 import json
-from collections import Counter
-from pathlib import Path
-
-import seaborn as sns
 import numpy as np
 from hypy_utils.scientific_utils import calc_col_stats
 from matplotlib import pyplot as plt
+from pathlib import Path
```

Comment:
Our linter checks .py files too?

Reply:
Lol oops I just used IntelliJ's import cleanup, and it cleaned up py imports as well